sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -296,35 +296,16 @@ const Lottery = (props: { listing: Listing }) => {
               <Button
                 variant="primary"
                 onClick={async () => {
-                  setLoading(true)
```

Comment:
We will still want the loading state

Reply:
oh nice call out! Sorry I shouldn't have fiddled with the react logic so much updated

to answer your question, yes it does! Or rather the work is all wrapped in a try/catch block with only `setCsvExportLoading()` switching from true to false being outside that try catch. you can peep that at `/src/lib/hooks.ts -> useCsvExport()`